<launch>
    <node name="driver" pkg="jaguar" type="diff_drive">
        <!-- CAN bus Configuration -->
        <param name="port" value="/dev/ttyUSB0"/>
        <param name="id_left"  value="2"/>
        <param name="id_right" value="3"/>
        <!-- Periodic Message Rates -->
        <param name="heartbeat" value="50"/> <!-- period in ms -->
        <param name="status"    value="50"/> <!-- period in ms -->
        <!-- Robot Model Parameters -->
        <param name="ticks_per_rev" value="800.000"/>
        <param name="wheel_radius"  value="0.127"/> <!-- meters -->
        <param name="robot_radius"  value="0.419"/> <!-- meters -->
    </node>
</launch>
